src/ingest_validation_tools/directory-schemas/seqfish-v0.yaml

```
@@ -1,5 +1,65 @@
 files:
   -
-    pattern: 'TODO'
-    description: 'Directory structure not yet specified.'
-    required: True
\ No newline at end of file
+    pattern: 'codebook/channel_[0-9]+\.(tsv|csv)'
+    description: 'Barcodes.'
+    required: True
+    is_qa_qc: False    
+  -
+    example: 'HybCycle_10/MMStack_Pos12.ome.tif'
+    pattern: 'HybCycle_[0-9]+/MMStack_Pos[0-9]+\.ome\.(tif|tiff)'
+    description: 'Raw OME-TIFF images with a variable number of field of views, and slices per tissue.'
+    required: True
+    is_qa_qc: True
+  -
+    example: 'initial_background/MMStack_Pos11.ome.tif'
+    pattern: 'initial_background/MMStack_Pos[0-9]+\.ome\.tif'
+    description: 'Raw OME-TIFF image of image without probes before imaging hybridizations; used for background and autofluorescence removal.'
+    required: True
+    is_qa_qc: True
+  -
+    example: 'final_background/MMStack_Pos11.ome.tif'
+    pattern: 'final_background/MMStack_Pos[0-9]+\.ome\.(tif|tiff)'
+    description: 'Raw OME-TIFF image of image without probes after all other hybridizations; used for background and autofluorescence removal.'
+    required: True
+    is_qa_qc: True
+  -
+    example: 'segmentation/MMStack_Pos3.ome.tif'
+    pattern: 'segmentation/MMStack_Pos[0-9]+\.ome\.(tif|tiff)'
+    description: 'Labels TIFF image for segmentation.'
+    required: True
+    is_qa_qc: False
+  -
+    example: 'segmentation/raw_images/dapi_Pos19.tif'
+    pattern: 'segmentation/raw_images/dapi_Pos[0-9]+\.(tif|tiff)'
+    description: 'DAPI TIFF image.'
+    required: True
+    is_qa_qc: False
+  -
+    example: 'segmentation/raw_images/membrane_Pos52.tif'
+    pattern: 'segmentation/raw_images/membrane_Pos[0-9]+\.(tif|tiff)'
+    description: 'Membrane TIFF image.'
+    required: False
+    is_qa_qc: False
+  -
+    pattern: 'omexml'
```

```suggestion
    pattern: 'omexml/.*'
```
Each file path is matched against a pattern, so this would only have allowed a single file called `omexml`. (There was an issue for directories #1025, but there hasn't been a call for that.)

---

Resolved.